gasCosts/gasCosts.js

```diff
@@ -313,7 +313,7 @@ contract("All", accounts => {
     });
 
     it("when working with reward payouts", async () => {
-      const totalReputation = toBN(350 * 1e18);
+      const totalReputation = toBN(300 * 1e18);
```

Comment:
You've changed this to be accurate, but I don't think this variable is used anywhere in the test?

Reply:
We are passing it to `claimRewardPayout`